lib/carma/models/submission.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module CARMA
+  module Models
+    class Submission < Base
+      attr_reader :metadata
+      attr_accessor :data
+      attr_accessor :carma_case_id, :submitted_at
```

Comment:
Nitpicky, but any reason not to include these all on one line?
```suggestion
      attr_accessor :carma_case_id, :data, :submitted_at
```